modules/rosapps/applications/sysutils/kill/kill.c

```diff
@@ -28,51 +28,47 @@
  * from the old shell.exe
  */
 
-#define WIN32_LEAN_AND_MEAN	/* Exclude rarely-used stuff from Windows headers */
+#define WIN32_LEAN_AND_MEAN /* Exclude rarely-used stuff from Windows headers */
 #include <windows.h>
+
 #include <stdio.h>
 #include <stdlib.h>
 
 int
-ExecuteKill(char * lpPid)
+ExecuteKill(const char *lpPid)
 {
-	HANDLE	hProcess;
-	DWORD	dwProcessId;
+    HANDLE hProcess;
+    DWORD dwProcessId;
+
+    dwProcessId = strtoul(lpPid, NULL, 10);
+    fprintf(stderr, "Killing the process with PID %lu...\n", dwProcessId);
```

Comment:
Nit: I see no reason to increase this strings length. Aside from that LGTM.

Reply:
Only to match the 2 other strings.

Reply:
I saw that, and still thought there is no reason to change. But that didn't prevent me giving my approval.